test/fixtures/v5/end_of_road/sharp_left_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Gire a cerrada a la izquierda",
         "fr": "Tourner franchement à gauche",
         "id": "Belok tajam kiri",
+        "it": "Gira a Leggera svolta a sinistra",
```

Comment:
This doesn't sound good. At present I do not know how to translate it better. I'll think about it.